cypress/integration/pages/home_page_spec.ts

```diff
@@ -0,0 +1,17 @@
+/// <reference types="cypress" />
+export {};
+
+describe('Test Home Page', () => {
+  it('loads successfully', () => {
+    cy.visit('/');
+  });
+  it('renders the hero title', () => {
+    cy.contains('Find, Share and Publish Quality Data with Datahub');
+  });
+  it('displays a search form', () => {
+    cy.get('form').type('dataset').contains('Search').click();
```

Comment:
Can we test if it actually works? E.g., it should redirect you to search page with results for your query.